pkgs/development/python-modules/pysequoia/default.nix

```diff
@@ -5,18 +5,12 @@
   pkg-config,
   rustPlatform,
   cargo,
-  rustc,
-  bzip2,
-  nettle,
-  openssl,
-  pcsclite,
   stdenv,
-  libiconv,
 }:
 
 buildPythonPackage rec {
   pname = "pysequoia";
-  version = "0.1.26";
```

Comment:
The hashes were not updated! Please use `nix-commit` to avoid this mistake.

Reply:
Sorry for the confusion. I tried looking up `nix-commit` but failed (I looked through google, github and in here: https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name/ni ).

Would you mind giving me an URL?

Reply:
Oh I'm sorry, I meant `nix-update` (Freudian slip when thinking about `nix-update --commit`). URL is:

https://github.com/Mic92/nix-update

Reply:
Okay, phew, it took me a bit to figure out but I did:

```bash
podman run --rm -it -v $PWD:$PWD -w $PWD ghcr.io/nixos/nix
nix-shell -p nix-update
nix-update python3.pkgs.pysequoia
```

and then reviewed what it did (updated hashes), squashed the thing and pushed... fingers crossed it works this time :crossed_fingers: